docs/infra/website.md

```diff
@@ -63,6 +63,51 @@ Other things happening.
 ```
 
 
+## Create a development environment
+
+This website uses [**containers**](https://web.archive.org/web/20200203111523/https://www.docker.com/resources/what-container) for a development environment.
+A container runtime like Docker or Podman works on Linux, macOS, and Windows.
+Containers closely mimic how the production version of the site is deployed.
+This means when you test your changes locally with containers, they will deploy in production successfully.
+This is also how maintainers test new pull requests.
+
+### Pre-requisites
+
+Install one of the following container runtimes:
+
+#### Docker
+
+* Linux
+  * [Fedora](https://developer.fedoraproject.org/tools/docker/docker-installation.html)
+  * [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+* [macOS](https://docs.docker.com/docker-for-mac/install/)
+<!-- * [Windows](https://docs.docker.com/toolbox/overview/) -->
+
+#### Podman
+
+* [Podman installation instructions](https://podman.io/getting-started/installation)
+
+### Build and serve Jekyll locally
+
+To build the site and serve it as a website from your workstation, run the following script:
+
+```sh
+./build.sh
+```
```

Comment:
I am a bit confused here. I can't find `build.sh` in `fossrit.github.io`
Would you mind adding the location of the script to run in order to build and serve Jekyll locally?

Reply:
Got it. I see build has not yet been push because of PR #88 (https://github.com/FOSSRIT/fossrit.github.io/pull/88).